include/cugraph.h

```diff
@@ -3,9 +3,9 @@
 #include <cstdlib>
 #include <cstdint>
 
-#include <gdf/gdf.h>
-#include <gdf/utils.h>
-#include <gdf/errorutils.h>
+#include <cudf.h>
+#include <cudf_utils.h>
+#include <error_utils.h>
```

Comment:
Has cuDF been patched to install this files?

Reply:
Not yet

Reply:
Hence the WIP